File: src/main/java/no/nav/service/pensjon/regler/to/RegulerBeregningMangelfulltGrunnlagRequest.java
package no.nav.service.pensjon.regler.to;

import java.util.Date;

import no.nav.domain.pensjon.regler.beregning.Beregning;
import no.nav.domain.pensjon.regler.beregning2011.AbstraktBeregningsResultat;

public class RegulerBeregningMangelfulltGrunnlagRequest extends ServiceRequest  {

	private Date virkFom;
	private Beregning beregning1967;
	private AbstraktBeregningsResultat beregningsResultat2011;

}

File: src/main/kotlin/no/nav/pensjon/regler/to/RegulerBeregningMangelfulltGrunnlagRequest.kt
package no.nav.pensjon.regler.to

import no.nav.pensjon.regler.domain.beregning.Beregning
import no.nav.pensjon.regler.domain.beregning2011.AbstraktBeregningsResultat
import java.util.*

@Deprecated("Avvikles. Bruk FaktorOmregnRequest.", ReplaceWith("no.nav.pensjon.regler.to.FaktorOmregnRequest"))
class RegulerBeregningMangelfulltGrunnlagRequest : ServiceRequest() {
    var virkFom: Date? = null
    var beregning1967: Beregning? = null
    var beregningsResultat2011: AbstraktBeregningsResultat? = null
}